src/system/applications/actor/components/character/connections-list.ts

```diff
@@ -106,45 +66,38 @@ any> {
         }, 50);
     }
 
-    public static async onRemoveConnection(
+    public static onRemoveConnection(
         this: CharacterConnectionsListComponent,
+        element: HTMLElement,
     ) {
-        this.controlsDropdownExpanded = false;
-
-        // Ensure context goal id is set
-        if (this.contextConnectionId !== null) {
-            // Remove the connection
-            await this.application.actor.deleteEmbeddedDocuments(
-                'Item',
-                [this.contextConnectionId],
-                { render: false },
-            );
+        const connectionId = $(element).closest('.item[data-id]').data('id') as
+            | string
+            | undefined;
```

Comment:
```suggestion
        const connectionId = element.querySelector('.item[data-id]')?.getAttribute('data-id');
```
Please can we start thinking about avoiding jQuery? This native line is just as good (and doesn't need a cast)